### PR TITLE
Make explicit a number of lines is required

### PR DESCRIPTION
**Pull request by jetm**
Make explicit a number of lines is required to comment.

### CODE REVIEW

**Comment by wsdjeg** [comment]
SPT  c t is not for comment next N number lines. when you input 10, It means comment comment form current line to line 10


so Line number maybe easy to understand. BTW, we maybe should not use relatenumber in this func.

**Comment by jetm** [comment]
Ohh, I was considering the output of relative number. I guess relative number should be disabled when this is called to avoid misunderstanding like this.

File: autoload/SpaceVim/mapping/space.vim [hunk 1]
```diff
@@ -81,8 +81,8 @@ function! SpaceVim#mapping#space#init() abort
 
   nnoremap <silent> <Plug>CommentToLine :call <SID>comment_to_line(0)<Cr>
   nnoremap <silent> <Plug>CommentToLineInvert :call <SID>comment_to_line(1)<Cr>
-  call SpaceVim#mapping#space#def('nmap', ['c', 't'], '<Plug>CommentToLine', 'comment to line', 0, 1)
-  call SpaceVim#mapping#space#def('nmap', ['c', 'T'], '<Plug>CommentToLineInvert', 'toggle comment to line', 0, 1)
+  call SpaceVim#mapping#space#def('nmap', ['c', 't'], '<Plug>CommentToLine', 'comment until the line', 0, 1)
+  call SpaceVim#mapping#space#def('nmap', ['c', 'T'], '<Plug>CommentToLineInvert', 'toggle until the line', 0, 1)
```
Comment (wsdjeg) [inline autoload/SpaceVim/mapping/space.vim]: hmm, toggle comment until the line.

Comment (jetm) [reply autoload/SpaceVim/mapping/space.vim]: Yes, I deleted by mistake. Thanks! New commit is coming...

**Comment by wsdjeg** [comment]
merged, thanks!

**Pull request merged**